PartsApp/DatabaseHelper/DataCheck.cs

```diff
@@ -0,0 +1,22 @@
+﻿using PartsApp.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PartsApp.DatabaseHelper
```

Comment:
it's not a DB layer it must be in business-layer

Reply:
corrected in PR 38 https://github.com/dmytroshchotkin/TruckTir/pull/38